manifests/install/charts/as-a-second-scheduler/README.md

```diff
@@ -26,7 +26,8 @@ Quick start instructions for the setup and configuration of as-a-second-schedule
 ```bash
 $ git clone git@github.com:kubernetes-sigs/scheduler-plugins.git
 $ cd scheduler-plugins/manifests/install/charts
-$ helm install scheduler-plugins as-a-second-scheduler/
+# you can specify target namespace by `--namespaces` option
+$ helm install scheduler-plugins as-a-second-scheduler/ --create-namespace --namespace scheduler-plugins
```

Comment:
I'd like to support --create-namespace and --namespace as a bonus, i.e., if these 2 parameters are not provided, the old command `helm install scheduler-plugins as-a-second-scheduler/` still functions well and defaults the namespace to `scheduler-plugins`. But now it defaults the current-namespace when user runs the command, which should be fixed.

cc @zwpaper

Reply:
let me raise a PR later today, we should be in time for v0.25.x

Reply:
@Huang-Wei I found a limitation in helm, `{{ .Release.Namespace }}`, which is the value of `-n` args in `helm install`, will be rendered to `default` no matter user omits the `-n` args or specifies `-n default`.

in short, we can not distinguish `-n default` from not specifying `-n` when using `helm install`.

how about we install the scheduler plugins to `scheduler-plugins` ns in both cases when specifying `-n default` or not specifying a ns?

Reply:
> will be rendered to `default` no matter user omits the `-n` args or specifies `-n default`

{{ .Release.Namespace }} is not always to be rendered to `default` namespace. It depends on the `default namspace` in kubectl current-context.
{{ .Release.Namespace }} will be rendered to `test` namespace after you set current context namespace to `test`:
```bash
k config set-context --current --namespace=test
```

Reply:
thanks for pointing that out @czybjtu, but it should be ok when users not specifying `-n`, no matter what the default ns is, we can install scheduler plugins in `scheduler-plugins` ns.

the problem is that when users specified `-n default`, can we still install in `scheduler-plugins` ns? 

helm chart can not detect whether `-n` is specified.

Reply:
Here is my 2 cents:

- `helm install`: the whole installation dumps stuff to ns `scheduler-plugins`, and it creates the namespace `scheduler-plugins` if needed. This is aligned with the old behavior.
- `helm install -n <ns>`: install to `<ns>`. If `<ns>` doesn't exist, the install fails.
- `helm install -n <ns> --create-namespace`: install to `<ns>` and create `<ns>` if it doesn't exist.

Reply:
> the problem is that when users specified -n default, can we still install in scheduler-plugins ns?

I'd say if `-n` is specified, we should install to that specified namespace.

Reply:
> helm chart can not detect whether -n is specified.

I found a way to work it around: check whether `.Release.Namespace` is `default`. But yes, in this case, we cannot support installing the chart to `default` namespace until Helm support a built-in variable like `Args`.

Will send out a PR shortly.

Reply:
I have reconsidered the installation method, and have a few different thoughts.

Let's step back to the why we want to raise this patch, we could keep the compatibility to the origin installation, but this not followed the helm way.

Helm chart is namespaced, it doesn't make that sense if we installed a chart under default ns, but all contents appear in another one, like `scheduler-plugins`.

I have checked some the other projects, like descheduler, nginx-ingress, they both encourage people to use their chart with namespace specified.

Ref: 
- https://kubernetes.github.io/ingress-nginx/deploy/#quick-start
- https://github.com/kubernetes-sigs/descheduler/blob/master/charts/descheduler/README.md

Reply:
So maybe we should reconsider whether we want to keep the previous installation way, or switch to the new one with namespace recommended

Reply:
@zwpaper as long as the new way is idiomatic, I'm fine with that. One question: is it conventional to save the efforts of creating namespace for the user? i.e., `helm install ... -n <ns>` should be good enough?

btw: the workaround I gave doesn't quite work, thanks @czybjtu for pointing out - the current-context matters.

Reply:
Good point, as I know, most cases do NOT create ns for users, we should drop the `--create-namespace` in the example, and then add a line to mentions that user could also create the ns in the same time with `--create-namespace`

@czybjtu